src/Sarif/Autogenerated/ReportingDescriptorEqualityComparer.cs

```diff
@@ -190,7 +190,7 @@ public bool Equals(ReportingDescriptor left, ReportingDescriptor right)
                         return false;
                     }
 
-                    if (!object.Equals(value_2.Value, value_3))
+                    if (!SerializedPropertyInfo.ValueComparer.Equals(value_2.Value, value_3))
```

Comment:
>if (!SerializedPropertyInfo.ValueComparer.Equals(value_2.Value, value_3)) [](http://example.com/codeflow?start=20&length=73)

just a note that when we change the NotYetAutoGenerated one we should always also change and checkin the AutoGenerated one to keep them in sync.

I just did a comparison of all currently what we have, this file is the only one not in sync.

Reply:
OK! So I will retain this file.